<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVQTImageToolbar</class>
 <widget class="QWidget" name="CVQTImageToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="backingLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="valueSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="enabledCheckbox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
